.env_SAMPLE

```diff
@@ -4,7 +4,21 @@
 # --------------------------------------------------------------------------
 
 ###############################
-# SHEER - for serving the site.
+ V1 - application feature related variables
```

Comment:
Also add the # to comment out this line and avoid `-bash: V1: command not found`